APMrover2/Log.cpp

```diff
@@ -284,11 +311,11 @@ void Rover::Log_Write_Attitude()
     DataFlash.Log_Write_Attitude(ahrs, targets);
 
 #if AP_AHRS_NAVEKF_AVAILABLE
- #if defined(OPTFLOW) and (OPTFLOW == ENABLED)
+#if defined(OPTFLOW) and (OPTFLOW == ENABLED)
```

Comment:
It's 3AM here, so maybe I'm sleeping already, but isn't this wrong? Can you use *and* instead of *&&*?

Also I really don't like the space removed. How is this more readable @lucasdemarchi (because you were the one saying you liked it this way)? When I looked at the else below I already didn't remembered it was inside another *if* (OK, OK, I'm exaggerating but you get my point).